bedrock/firefox/views.py

```diff
@@ -250,17 +249,15 @@ def all_downloads(request, channel):
     return l10n_utils.render(request, 'firefox/all.html', context)
 
 
-def firefox_os_index(request):
-
-    locale = l10n_utils.get_locale(request)
-    lang_file = 'firefox/os/index-new'
-    old_home = 'firefox/os/index.html'
-    new_home = 'firefox/os/index-new.html'
+@never_cache
+def firefox_os_geo_redirect(request):
+    country = get_country_from_request(request)
+    try:
+        version = settings.FIREFOX_OS_COUNTRY_VERSIONS[country]
+    except KeyError:
+        version = settings.FIREFOX_OS_COUNTRY_VERSIONS['default']
```

Comment:
NIT: I personally prefer the `get` method with a default over exception handling. How about something like this instead?

```python
version = settings.FIREFOX_OS_COUNTRY_VERSIONS.get(
    country,
    settings.FIREFOX_OS_COUNTRY_VERSIONS['default'])
```